src/ApiService/ApiService/onefuzzlib/DiskOperations.cs

```diff
@@ -19,8 +20,18 @@ public class DiskOperations : IDiskOperations {
         _creds = creds;
     }
 
-    public Task<bool> DeleteDisk(string resourceGroup, string name) {
-        throw new NotImplementedException();
+    public async Task<bool> DeleteDisk(string resourceGroup, string name) {
+        try {
+            _logTracer.Info($"deleting disks {resourceGroup} : {name}");
+            var disk = await _creds.GetResourceGroupResource().GetDiskAsync(name);
+            if (disk != null) {
+                await disk.Value.DeleteAsync(WaitUntil.Started);
+                return true;
+            }
+        } catch (Exception e) {
+            _logTracer.Error($"unable to delete disk: {name} {e.Message}");
```

Comment:
there is also _logTracer.Exception(e)
(better to do both Error and Exception (or maybe modify log tracer to do both when Exception is logged...)